microsoft/testsuites/performance/nvmeperf.py

```diff
@@ -72,3 +72,52 @@ def perf_nvme(self, node: Node, result: TestResult) -> None:
             disk_type=DiskType.nvme,
             test_result=result,
         )
+
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test NVMe disk performance
+        with 'io_uring' as ioengine.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            supported_features=[NvmeSettings(disk_count=8)],
```

Comment:
The test case should be skipped, if it's not Nvme disk. Please make sure it works like that.

Reply:
The logic is already in the testcase.

Reply:
1. The requirements can find vm size with nvme by feature implementation. If the checks are in test case, it may not match to right vm size for general distro testing.
2. In the test case, I see assertion, but no skipped exception. It makes case failed, instead of skip for unmatched env.

Reply:
Changed the requirement to supported_features=[Nvme].